### PR TITLE
Remove org.eclipse.xtext.generator consumers from DDK

### PR DESCRIPTION
**Pull request by KrisLimbo**
Removed org.eclipse.xtext.generator in required-bundle entry in
MANIFEST.MF.

Removed org.eclipse.xtext.generator import in MWE2 workflow files.

Updated the following to use non-deprecated classes:
CheckGenModelUtil, GenModelUtil2, ParserSemanticPredicatesUtil
FormatGenerator, FormatUiModule

Removed .XPT files and fragments using deprecated classes in
org.eclipse.xtext.generator.

Run DDK.MWE2.

### CODE REVIEW

File: com.avaloq.tools.ddk.sample.helloworld/dsl-gen/com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironmentStandaloneSetup.java
Comment (rubenporras) [inline com.avaloq.tools.ddk.sample.helloworld/dsl-gen/com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironmentStandaloneSetup.java:21]: The annotation /** {@inheritDoc} */ is not correct. If it is an override, there is nothing to inherit for the documentation. Could you remove that from this place and the rest in this PR?

File: com.avaloq.tools.ddk.workflow/src/com/avaloq/tools/ddk/workflow/GenerateFormatterTestLanguage.mwe2
Comment (rubenporras) [inline com.avaloq.tools.ddk.workflow/src/com/avaloq/tools/ddk/workflow/GenerateFormatterTestLanguage.mwe2:2]: this change in copyright is not correct, could you revert it here and in the rest of the PR?

File: com.avaloq.tools.ddk.workflow/src/com/avaloq/tools/ddk/workflow/TypeModel.mwe2
Comment (rubenporras) [inline com.avaloq.tools.ddk.workflow/src/com/avaloq/tools/ddk/workflow/TypeModel.mwe2:20]: remove?

File: com.avaloq.tools.ddk.xtext.expression/src/com/avaloq/tools/ddk/xtext/expression/generator/GenModelUtil2.java
Comment (rubenporras) [inline com.avaloq.tools.ddk.xtext.expression/src/com/avaloq/tools/ddk/xtext/expression/generator/GenModelUtil2.java:122]: we should use import as usual

File: com.avaloq.tools.ddk.check.core/src/com/avaloq/tools/ddk/check/generator/CheckGenModelUtil.java
Comment (rubenporras) [inline com.avaloq.tools.ddk.check.core/src/com/avaloq/tools/ddk/check/generator/CheckGenModelUtil.java:112]: we should use import as usual

File: com.avaloq.tools.ddk.xtext.format.test/build.properties
Comment (rubenporras) [inline com.avaloq.tools.ddk.xtext.format.test/build.properties:4]: why the change?

File: com.avaloq.tools.ddk.xtext.format/model/generated/Format.ecore
Comment (rubenporras) [inline com.avaloq.tools.ddk.xtext.format/model/generated/Format.ecore:6]: this changes are not correct. Could we revert the file?

File: com.avaloq.tools.ddk.xtext.format/model/generated/Format.genmodel
Comment (rubenporras) [inline com.avaloq.tools.ddk.xtext.format/model/generated/Format.genmodel:3]: this changes are not correct. Could we revert the file?

**Review by rubenporras — CHANGES_REQUESTED** [review]

File: com.avaloq.tools.ddk.xtext.expression/src/com/avaloq/tools/ddk/xtext/expression/generator/GenModelUtil2.java
Comment (KrisLimbo) [reply com.avaloq.tools.ddk.xtext.expression/src/com/avaloq/tools/ddk/xtext/expression/generator/GenModelUtil2.java:122]: I am using the qualified name because this class is referenced under a class of the same name.
Do you want me to rename GenModelUtil3 here? I am not sure if it is safe to do so.

File: com.avaloq.tools.ddk.sample.helloworld/dsl-gen/com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironmentStandaloneSetup.java
Comment (KrisLimbo) [reply com.avaloq.tools.ddk.sample.helloworld/dsl-gen/com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironmentStandaloneSetup.java]: Ok.

File: com.avaloq.tools.ddk.workflow/src/com/avaloq/tools/ddk/workflow/GenerateFormatterTestLanguage.mwe2
Comment (KrisLimbo) [reply com.avaloq.tools.ddk.workflow/src/com/avaloq/tools/ddk/workflow/GenerateFormatterTestLanguage.mwe2]: Ok.

File: com.avaloq.tools.ddk.workflow/src/com/avaloq/tools/ddk/workflow/TypeModel.mwe2
Comment (KrisLimbo) [reply com.avaloq.tools.ddk.workflow/src/com/avaloq/tools/ddk/workflow/TypeModel.mwe2]: Ok.

File: com.avaloq.tools.ddk.check.core/src/com/avaloq/tools/ddk/check/generator/CheckGenModelUtil.java
Comment (KrisLimbo) [reply com.avaloq.tools.ddk.check.core/src/com/avaloq/tools/ddk/check/generator/CheckGenModelUtil.java]: done.

File: com.avaloq.tools.ddk.xtext.format.test/build.properties
Comment (KrisLimbo) [reply com.avaloq.tools.ddk.xtext.format.test/build.properties]: probably just a local workspace change. reverted.

File: com.avaloq.tools.ddk.xtext.format/model/generated/Format.ecore
Comment (KrisLimbo) [reply com.avaloq.tools.ddk.xtext.format/model/generated/Format.ecore]: reverted. thanks!

File: com.avaloq.tools.ddk.xtext.format/model/generated/Format.genmodel
Comment (KrisLimbo) [reply com.avaloq.tools.ddk.xtext.format/model/generated/Format.genmodel]: reverted.

**Review by rubenporras — APPROVED** [review]

**Comment by rubenporras** [comment]
@KrisLimbo , please wait for the release next week before merging

**Pull request merged**
